Assets/Scripts/UI/Menu's/WinScreenController.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public sealed class WinScreenController : MonoBehaviour
+{
+    [SerializeField] private GameObject player;
+
+    private bool _isPickedUp;
```

Comment:
```suggestion
    private bool _canWin;
```
Dit script heeft niks met de pickup naming te maken.